test/Mono.Linker.Tests.Cases/Warnings/WarningSuppression/SuppressWarningsInCompilerGeneratedCode.cs

```diff
@@ -45,7 +45,8 @@ static IEnumerable<int> TestReflectionAccessRUCMethod ()
 			static IEnumerable<int> TestLdftnOnRUCMethod ()
 			{
 				yield return 0;
-				var _ = new Action (RequiresUnreferencedCodeMethod);
+				var a = new Action (RequiresUnreferencedCodeMethod);
```

Comment:
Maybe we can create an additional test with the discard, analyzer handles it in a particular way. It would be interesting to have a test case with a discard for later

Reply:
I forgot to revert testing changes